src/selectors/character.ts

```diff
@@ -2,15 +2,28 @@ import {createSelector} from 'reselect';
 import {formStateSelector} from './form';
 import {IForm} from '../store/form/types';
 
+const validRaceAlignments = {
+  tiefling: ['Chaotic Neutral', 'Neutral', 'Chaotic Evil', 'Neutral Evil'],
+  human: ['Lawful Good', 'Neutral Good', 'Chaotic Good', 'Neutral', 'Lawful Neutral', 'Neutral Evil', 'Lawful Evil'],
+  elf: ['Neutral', 'Neutral Good', 'Lawful Neutral', 'Lawful Good'],
+};
+
 const characterFormSelector = createSelector(
   formStateSelector,
   (form: IForm) => form.character
 );
 
+const isRaceAlignmentValid = (character) => {
+  const race = character.bioSummary.race.toLowerCase();
+  const valid = validRaceAlignments[race].filter(value => value === character.bioSummary.alignment);
```

Comment:
Small thing but this is better written with a find
Anyhow this selector looks good always computing stuff from the state instead of saving the computation there